src/Nordea/Components/MultiSelectDropdown.elm

```diff
@@ -170,7 +170,7 @@ view attrs dropdown =
     in
     Html.column attrs
         [ Html.row [ css [ displayFlex, justifyContent spaceBetween, marginBottom (rem 0.2) ] ]
-            [ Label.init { label = dropdown.label } |> Label.view []
+            [ Label.init { label = dropdown.label } |> Label.withAsLegend |> Label.view []
```

Comment:
I suspect a `<fieldset>` tag is required as a wrapper for `legend` to be semantically connected 🤔

Reply:
Good catch! I'll add it 🙌